js/piwik.js

```diff
@@ -1138,15 +1138,7 @@ if (typeof Piwik !== 'object') {
             }
 
             function _elementInDocument(element) {
-                element = element.parentNode;
-
-                while (element) {
-                    if (element === documentAlias) {
-                        return true;
-                    }
-                    element = element.parentNode;
-                }
-                return false;
+                return documentAlias.documentElement.contains(element);
```

Comment:
Thx! It's also supported since Firefox 9.0. @mattab do we need to support older versions? I'm not sure re our policy here.